server/apps/duplication/archive_move.py

```diff
@@ -73,15 +70,15 @@ def move_content(self, id, doc):
         if current_stage_of_item and str(current_stage_of_item) == str(doc.get('task', {}).get('stage')):
             raise SuperdeskApiError.preconditionFailedError(message='Move is not allowed within the same stage.')
 
-        if not is_workflow_state_transition_valid('submit_to_desk', archived_doc[config.CONTENT_STATE]):
+        if not is_workflow_state_transition_valid('submit_to_desk', archived_doc[ITEM_STATE]):
             raise InvalidStateTransitionError()
 
         original = dict(archived_doc)
 
         send_to(doc=archived_doc, desk_id=doc.get('task', {}).get('desc'), stage_id=doc.get('task', {}).get('stage'))
 
-        if archived_doc[config.CONTENT_STATE] not in ['published', 'scheduled', 'killed']:
-            archived_doc[config.CONTENT_STATE] = 'submitted'
+        if archived_doc[ITEM_STATE] not in [CONTENT_STATE.PUBLISHED, CONTENT_STATE.SCHEDULED, CONTENT_STATE.KILLED]:
```

Comment:
(minor) Same here, a set (`{first, second, third}`) is a more suitable data structure.